addons/point_of_sale/static/tests/pos/tours/utils/product_screen_util.js

```diff
@@ -844,3 +844,20 @@ export function createProductFromFrontend(name, barcode, list_price, category) {
 export function editProductFromFrontend(name, barcode, list_price) {
     return productInputSteps(name, barcode, list_price);
 }
+
+export function longPressProduct(productName) {
+    return [
+        {
+            content: `Long pressing product "${productName}"...`,
+            trigger: `.product-name:contains("${productName}")`,
+            run: async () => {
+                const el = document.querySelector(".product-name");
+                const mouseDown = new MouseEvent("mousedown", { bubbles: true });
+                const mouseUp = new MouseEvent("mouseup", { bubbles: true });
+                el.dispatchEvent(mouseDown);
+                await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
Can you make the longpress duration to be only 250ms when in testmode? Perhaps put it in the `@point_of_sale/utils` module and compute it based on the `@web/session`'s `test_mode`.

Reply:
Also, I think you should add a few milliseconds to ensure that the mouse-up event is triggered at the right time. Perhaps add 50ms.